specs/phase0/beacon-chain.md

```diff
@@ -311,6 +320,8 @@ class Checkpoint(Container):
 
 #### `Validator`
 
+All relevant about a validator.
+
```

Comment:
Maybe a comment here about the balances being stored separately (for merkleization / change-control purposes) is good? It's does not include "all" about a validator here.